app/controllers/admins_controller.rb

```diff
@@ -13,6 +13,13 @@ def approve
         @item.save
         redirect_to admins_path
     end
+    
+    def approve_p
```

Comment:
All of the `method` and `method_p` definitions seem quite repetitive. Is there a way that you could DRY these out?

Reply:
All removed except approve and approve_p.  The other methods are handled in their respective controllers.